Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -45,7 +45,7 @@ public float groundedMoveSpeed
 	// Slope limit
 
 	protected float groundSlopeSpeedMod = 1f;
-	[SerializeField] MinMax slopeLimits = new MinMax( 0.28f, 0.72f );
+	[SerializeField] MinMaxF slopeLimits = new MinMaxF( 0.28f, 0.72f );
```

Comment:
Make sure that this can be set in the inspector. It looks like it can but I'm not sure.